src/views/IndexView.jsx

```diff
@@ -1,11 +1,45 @@
-import React from 'react'
+import React from "react"
+import LogoCHWL from "../assets/LogoCHWL.png"
+import { Grid, Box } from "@material-ui/core"
 
 const IndexView = () => {
   return (
     <>
-      <h3 data-cy="header-subtitle">Self care for better health</h3>
+      <Grid container justifyItems="center">
+        <Grid item xs={12} lg={6}>
+          <Box component="div" style={styles.gridItem}>
+            <img
+              src={LogoCHWL}
+              data-cy="logo"
+              style={styles.image}
+              alt="Community Health West London"
+            />
+            <h3 data-cy="mission-statement" style={styles.statement}>
+              Our aim is to improve people's health and well-being
+            </h3>
+          </Box>
+        </Grid>        
+      </Grid>
```

Comment:
You can remove the Fragment wrapping `Grid`

Reply:
done